Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "ImPlotWidgets/Public/LineFilledGraphWidget.h"
+#include "ImGui/Public/ImPlotImplementation.h"
+
+void SFilledGraphWidget::Construct(const FArguments& InArgs)
+{
+	InData = InArgs._InDataArray;
+
+	ChildSlot
+		[
+			SAssignNew(ImGuiCustomWidget, SImGuiCustomWidget)
+			.ContextIndex(InArgs._ContextIndex)
+			.OnImGuiDrawDelegate(this, &SFilledGraphWidget::OnImGuiDraw)
+		];
+}
+
+void SFilledGraphWidget::OnImGuiDraw() const
+{
+	constexpr int32 WindowFlags = ImGuiWindowFlags_NoDecoration
+		| ImGuiWindowFlags_NoBackground
+		| ImGuiWindowFlags_NoSavedSettings;
+
+	constexpr int32 GraphFlags = ImPlotFlags_CanvasOnly | ImPlotFlags_AntiAliased;
+
+	constexpr int32 AxisFlags = ImPlotAxisFlags_NoDecorations;
+
+	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
+
+	ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
+	ImGui::SetNextWindowSize(ImVec2(AllottedGeometry.GetLocalSize().X, AllottedGeometry.GetLocalSize().Y), ImGuiCond_Always);
+	ImGui::Begin("ImPlot Demo", NULL, WindowFlags);
+
+	// ---------------------------------
+	{
+		TArray<float> XValues;
+		for (float f = 0; f < InData.Num(); ++f)
```

Comment:
I wonder if you could use std::iota here? https://en.cppreference.com/w/cpp/algorithm/iota

(Single letter var names should be avoided so can you make 'f' something clearer please!)